### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA are not announcing the listitems present under comboxes in "Run on web app" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1462]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on web app
4. "Run on  web app" dialog will open. 
5. In "Run on web app" dialog , tab to  and select "Create new web app" radio buttons.
6. Tab to Tab to Subscription combo box 
    "use Existing" combo box under resource group expand button.
    "Location and Pricing Tier" combo boxes under "App Service Plan" expand button.
    "Java Version and Artifact" combo boxes under "Java" expand button 
    and use alt+down arrow to expand the combo box
7. Observe the behaviour of Jaws and NVDA 

**Actual Result:**
List items under combo boxes are not announced by Jaws and NVDA

**Jaws behaviour:**
Combo box closed
1 of 23
To change the selection use the arrow keys
Open list Box
open
blank
blank
blank

**NVDA Behaviour:**
combo box  collapsed
list
list item  selected  1 of 23
list item  selected  2 of 23
list item  selected  3 of 23

**Expected Result:**
List items under combo box should be announced by Jaws and NVDA

Jaws should announce as
Use Existing Combo box closed
1 of 23
To change the selection use the arrow keys
Open list Box
open
Terraform TestRG list item  selected  1 of 23
abode RG list item  selected  2 of 23
anu-rg-eh-np list item  selected  3 of 23

NVDA should announce as
Use Existing combo box  collapsed
list
Terraform TestRG list item  selected  1 of 23
abode RG list item  selected  2 of 23
anu-rg-eh-np list item  selected  3 of 23

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on we app-combo box-list items](https://user-images.githubusercontent.com/22956130/36101560-b61219b8-102f-11e8-8439-bec03c0ba2d5.jpg)
![2ddd95f3](https://user-images.githubusercontent.com/22956130/36101576-c0efba02-102f-11e8-9dc9-94721cfd6a31.jpg)
![run on web app-listitems-explorer](https://user-images.githubusercontent.com/22956130/36101580-c500f700-102f-11e8-8de5-ddcb43bcb18c.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749930
